File: Activity48.c
#include <stdio.h>

int main() {
    double amount, futureValue;
    int years, i;               
    
    printf("Input initial amount: ");
    scanf("%lf", &amount);

    printf("Input number of years: ");
    scanf("%d", &years);
    
    if (amount <= 0 || years <= 0) {
        printf("Both amount and number of years must be positive.\n");
        return 0;
    }

    futureValue = amount;
    for (i = 0; i < years; i++) {
        futureValue = futureValue + (futureValue * 10 / 100); // add 10% each year

    }
    printf("After %d year(s)\nThe amount will be worth: %.2lf", years, futureValue);
    
    return 0;
}

File: Java/Activity48.java
import java.util.Scanner;

public class Activity48 {

    public static void main(String[] args) {

        Scanner input = new Scanner(System.in);

        double principal = -1;
        int years = -1;
        double rate = 0.10; 

        while (principal <= 0) {
            System.out.print("Enter the principal amount (positive number): ");
            principal = input.nextDouble();
            if (principal <= 0) {
                System.out.println("Amount must be positive. Try again.");
            }
        }

        while (years <= 0) {
            System.out.print("Enter the number of years (positive integer): ");
            years = input.nextInt();
            if (years <= 0) {
                System.out.println("Number of years must be positive. Try again.");
            }
        }

        double finalAmount = principal;
        for (int i = 1; i <= years; i++) {
            finalAmount = finalAmount + finalAmount * rate; 
        }

        System.out.printf("After %d years at 10%% interest, your investment will be worth %.2f%n", years, finalAmount);

        input.close();
    }
}
